backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -66,6 +83,69 @@ const appendFavourites = (req, res) => {
 const establishRoutes = () => {
 	const router = Router({ mergeParams: true });
 
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers:
+	 *   post:
+	 *     description: Add a new container to the specified project the user is admin of
+	 *     tags: [Containers]
+	 *     operationId: addContainer
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: Name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: Project ID
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     requestBody:
+	 *       content:
+   	 *         application/json:
+   	 *           schema:
+	 *             type: object
+	 *             properties:
+	 *               name:
+	 *                 type: string
+	 *               unit:
+	 *                 type: string
+	 *               desc:
+	 *                 type: string
+	 *               code:
+	 *                 type: string
+	 *               type:
+	 *                 type: string
+   	 *           example:
+	 *             name: Awesome Model
+	 *             unit: mm
+	 *             desc: This is an awesome model
+	 *             code: awe12
+	 *             type: Mechanical
+	 *     responses:
+	 *       401:
+	 *         $ref: "#/components/responses/notLoggedIn"
+	 *       404:
+	 *         $ref: "#/components/responses/projectNotFound"
+	 *       200:
+	 *         description: Container ID
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               type: object
+	 *               properties:
+	 *                 _id:
+	 *                   type: string
```

Comment:
specify the format. https://swagger.io/docs/specification/data-models/data-types/#string